apps-rendering/src/components/list.tsx

```diff
@@ -23,14 +29,28 @@ const styles: SerializedStyles = css`
 		background-color: ${neutral[86]};
 		margin-left: -${remSpace[6]};
 		vertical-align: middle;
+
+		${darkModeCss`
+		${
+			isStandfirst && format
+				? `background-color: ${background.liveblogStandfirstBulletDark(
```

Comment:
I think you could just call this `standfirstBulletDark`? The liveblog part of it is taken care of by the presence of the `format` argument?